css/style.css

```diff
@@ -1,743 +1,760 @@
 /* BOX SIZING */
-html {
```

Comment:
Just moving this down a few lines after the variables

Reply:
Awesome pawsome!